src/converters.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  EntityFromIntegration,
+  RelationshipFromIntegration
+} from "@jupiterone/jupiter-managed-integration-sdk";
+import { Agent, WazuhManager } from "./provider";
+
+export const ACCOUNT_ENTITY_TYPE = "provider_account";
+export const ACCOUNT_ENTITY_CLASS = "Account";
+
+export const WAZUH_MANAGER_ENTITY_TYPE = "provider_wazuhmanager";
+export const WAZUH_MANAGER_ENTITY_CLASS = "Wazuhmanager";
```

Comment:
Entities should have a class defined in the [data model](http://docs.jupiterone.io/en/latest/docs/jupiterone-data-model.html). I'd stick with just `Account` for now.

```suggestion
export const WAZUH_MANAGER_ENTITY_CLASS = "Account";
```